rsts/deployment/cluster_config/customizable_resources.rst

```diff
@@ -86,4 +86,107 @@ Flytekit
 
 For convenience, add a FlyteCTL wrapper to update the new attributes. Refer to `this PR <https://github.com/flyteorg/flytectl/pull/65>`__ for the entire set of changes required.
 
-That's it! You now have a new matchable attribute to configure as the needs of your users evolve.
\ No newline at end of file
+That's it! You now have a new matchable attribute to configure as the needs of your users evolve.
+
+Flyte ResourceManager
+---------------------
+
+**Flyte ResourceManager** is a configurable component that allows plugins to manage resource allocations independently. It helps track resource utilization of tasks that run on Flyte. The default deployments are configured as ``noop``, which means they are disabled and you would fallback to the platform specific resource management, like K8s, which tells you that you can't schedule more tasks.
```

Comment:
Can we elaborate on where the resource manager is beneficial? Maybe changing this portion:

"which means they are disabled and you would fallback to the platform specific resource management, like K8s, which tells you that you can't schedule more tasks."

To something like:

"which indicates the ResourceManager provided by Flyte is disabled and plugins rely on each independent platform to manage resource utilization. In situations like the k8s plugin, where the the platform has a robust mechanism to manage resource scheduling this may work well. However, in a scenario like a simple web API plugin, the rate at which Flyte sends requests may overwhelm a service and benefit from additional resource management."